config/mail.php

```diff
@@ -32,53 +32,46 @@
     |            "postmark", "log", "array", "failover"
     |
     */
-
+    
     'mailers' => [
         'smtp' => [
             'encryption' => env('MAIL_ENCRYPTION', 'tls'),
-            'host'       => env('MAIL_HOST', 'smtp.mailgun.org'),
-            'password'   => env('MAIL_PASSWORD'),
-            'port'       => env('MAIL_PORT', 587),
-            'timeout'    => null,
-            'transport'  => 'smtp',
-            'username'   => env('MAIL_USERNAME'),
+            'host' => env('MAIL_HOST', 'smtp.mailgun.org'),
+            'password' => env('MAIL_PASSWORD'),
+            'port' => env('MAIL_PORT', 587),
+            'timeout' => null,
+            'transport' => 'smtp',
+            'username' => env('MAIL_USERNAME'),
```

Comment:
@jaxwilko just to confirm, is trailing commas enforced in the ArrayPrinter?

Reply:
It was at some point yes, if you check the early posts on the PR there was an example of it, it'd be worth looking at the tests to confirm though

Reply:
Yes, it is